### ISSUE TITLE
More usage examples?

### ISSUE THREAD

**Issue by dmitriz** [JAForbes/pointer-stream#3]
It would be nice to give some examples 
of complete functional code snippets to demonstrate the features.

The readme example only shows how to subscribe but 
not how to actually drive the example to see some effects in browser.

**Comment by JAForbes** [comment]
Yeah great idea, like live demos?  What sort of thing would you like / expect to see?
